resources/views/categories/index.blade.php

```diff
@@ -0,0 +1,91 @@
+@extends('layouts.app')
+@section('page-title')
+  @lang('categories.index.title')
+@endsection
+
+@section('section-title')
+    @lang('categories.index.section-title')
+@endsection
+
+@section('page-content')
+    <div class="x_content">
+        <table class="table table-striped jambo_table table-bordered">
+          <thead>
+            <tr class="headings">
+              <th class="column-title text-center" style="width: 5%">#</th>
```

Comment:
[rank B] why didn't use datatable for this list? And we need to avoid using css inline.